packages/data/src/onboarding/reducer.js

```diff
@@ -28,24 +28,30 @@ export const defaultState = {
 	paymentMethods: [],
 	productTypes: [],
 	requesting: {},
-	taskLists: [],
+	taskLists: {},
```

Comment:
Updated this to be a key/value store to make retrieving individual task lists easier and also make it easier if we are requesting individual task lists.